Headers/headless/HeadlessContext.h

```diff
@@ -0,0 +1,28 @@
+/* HeadlessContext.c - Headless Context
+ * 
+ *  Copyright (c) 2023 Keysight Technologies
```

Comment:
I am a bit confused by this copyright line. Do we really want to keep this especially as you removed every content from the original files?

Overall I think you should go through the file headers once more and adjust these as needed..